mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -556,11 +556,38 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 						"enable_confidential_compute": {
 							Type:        schema.TypeBool,
 							Required:    true,
+							ForceNew:    true,
```

Comment:
@ScottSuarez @upodroid: Why was this made `ForceNew`? That's appears to be a breaking change at first glance, and I can't see any context in this PR on why it wouldn't be in this circumstance.

Reply:
It was something I forgot when I first added CVM to this resource. The resource doesn't have have an update method so it is safe to sneak this change through.

Reply:
Ack, thanks. If you're making similar changes in the future can you make sure to specifically call that out when you send the review? Showing that modifying the field always returns an error is quick to collect and does all we need to justify the change.

One case that gives me pause as a reviewer is when Terraform *doesn't* return an error. We may want to avoid the change in those cases, even if it's correct. Unfortunately not all users run `terraform plan` before apply, particularly after provider updates, and depending on the circumstances we may want to not make the change until a major version.

Reply:
I'll bear that in mind. There isn't a fix for end users running apply carelessly and not pinning provider versions.